src/browser/modules/Sidebar/About.tsx

```diff
@@ -142,19 +158,7 @@ const About = ({ serverVersion, serverEdition }) => (
     <DrawerFooter>With &#9829; from Sweden.</DrawerFooter>
   </Drawer>
 )
-
-const asChangeLogUrl = serverVersion => {
-  if (!serverVersion) {
-    return undefined
-  }
-  const tokenisedServerVersion = serverVersion && serverVersion.split('.')
-  const releaseTag = tokenisedServerVersion.join('')
-  const urlServerVersion =
-    serverVersion && tokenisedServerVersion.splice(0, 2).join('.')
-  return `https://github.com/neo4j/neo4j/wiki/Neo4j-${urlServerVersion}-changelog#${releaseTag}`
-}
-
-const mapStateToProps = state => {
+const mapStateToProps = (state: any) => {
```

Comment:
Getting a better solution to state types didn't seem trivial. It'll be something to look into in the future